mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -252,7 +252,7 @@ def create(self, validated_data):
         )
         db_helpers.upsert_event_records(event_records, "push", on_conflict_update=True)
 
-        # We don't have the created event records, but we'll return an empty response anyway
+        # We don't have the created event records, return an empty response anyway
```

Comment:
I think this could change the meaning, I wanted to emphasize the fact that we're using `EmptyReponseSerializer`, and it obviously doesn't care.